pkg/secrethub/credentials/encoding.go

```diff
@@ -22,9 +22,10 @@ var (
 	ErrInvalidCredentialHeaderField      = errCredentials.Code("invalid_credential_header_field").ErrorPref("invalid header field: %s")
 	ErrCannotDecodeCredentialHeader      = errCredentials.Code("invalid_credential_header").ErrorPref("cannot decode credential header: %v")
 	ErrUnsupportedCredentialType         = errCredentials.Code("unsupported_credential_type").ErrorPref("unsupported credential type: %s")
-	ErrCannotDecodeCredentialPayload     = errCredentials.Code("invalid_credential_header").ErrorPref("cannot decode credential payload: %v")
+	ErrCannotDecodeCredentialPayload     = errCredentials.Code("invalid_credential_payload").ErrorPref("cannot decode credential payload: %v")
```

Comment:
Did you intend to include this change? It seems unrelated to this PR? Do we want to change this error code and why?

Reply:
I think it was a mistake. I changed it back